ArduPlane/tiltrotor.cpp

```diff
@@ -516,9 +516,9 @@ void Tiltrotor::vectoring(void)
                 // base the tilt on elevon mixing, which means it
                 // takes account of the MIXING_GAIN. The rear tilt is
                 // based on elevator
-                const float right = gain * SRV_Channels::get_output_scaled(SRV_Channel::k_elevon_right) / 4500.0;
-                const float left  = gain * SRV_Channels::get_output_scaled(SRV_Channel::k_elevon_left) / 4500.0;
-                const float mid  = gain * SRV_Channels::get_output_scaled(SRV_Channel::k_elevator) / 4500.0;
+                const float right = gain * SRV_Channels::get_output_scaled(SRV_Channel::k_elevon_right) * 0.0002222; // 0.0002222 = 1/4500
```

Comment:
Just include the constant you want as an actual floating point calculation. This is very hard to read, and arbitrarily limits accuracy of the result due to truncation of the provided constant.